### ISSUE TITLE
evolutions not randomized

### ISSUE THREAD

**Issue by Gobl943** [Aldo796/BDSP-Randomizers#50]
Hey, don't know if this has been said before but the latest build doesn't have evolutions randomized even if I select the tab to do so. Even after re-randomizing, they aren't random.

**Comment by Ai0796** [comment]
do other randomizations work?

**Comment by Gobl943** [comment]
Yes. Just not evolutions. Even when the prompt is selected when randomizing.

**Comment by Zauke07** [comment]
I have no idea if they're still working on it, but unfortunately it still doesn't work. Without this function, the game is only half as much fun. Does anyone have a solution for this?
